### PR TITLE
support multiple columns in `.orderBy()`

### PR DESCRIPTION
**Pull request by HurSungYun**
implementation of #2874

### CODE REVIEW

File: src/query/builder.js [hunk 1]
```diff
@@ -549,6 +552,28 @@ assign(Builder.prototype, {
...
+    for (let i = 0; i < columnDefs.length; i++) {
+      const columnInfo = columnDefs[i];
+      if (isObject(columnInfo)) {
+        this._statements.push({
+          grouping: 'order',
+          type: 'orderByBasic',
+          value: columnInfo['column'],
+          direction: columnInfo['order'],
+        });
+      } else if (typeof columnInfo === 'string') {
```
Comment (kibertoad) [inline src/query/builder.js]: why not use _.isString?

File: src/query/builder.js [hunk 2]
```diff
@@ -549,6 +552,28 @@ assign(Builder.prototype, {
     return this;
   },
 
+  // Adds a `order by` with multiple columns to the query.
+  orderByArray(columnDefs) {
```
Comment (kibertoad) [inline src/query/builder.js]: is this supposed to be a public method?

**Comment by kibertoad** [comment]
Thank you for your contribution! Would you consider creating documentation in https://github.com/knex/documentation for it?

**Comment by kibertoad** [comment]
LGTM! @elhigu what do you think?

**Comment by elhigu** [comment]
Everything seemed fine to me 👍

**Pull request merged**